src/views/InvolvementsAll/components/Requests/index.jsx

```diff
@@ -46,7 +46,7 @@ const Requests = ({ profile, session }) => {
   } else if (involvementsLeading?.length > 0) {
     content = (
       <>
-        <CardHeader title="Membership Requests" className={styles.requests_header} />
+        {/* <CardHeader title="Membership Requests" className={styles.requests_header} /> */}
 
         <CardContent>
```

Comment:
Since you got rid of the surrounding `Card` component, you can remove the `CardContent` as well.

Reply:
Also, it looks like you need to remove the `CardHeader` and `CardContent` in the `else` block for when no requests are found (~lines 108, 115).